testdb.go

```diff
@@ -56,6 +56,11 @@ type Config struct {
 	// capabilities of this role should match the capabilities of the role that
 	// your application uses to connect to its database and run migrations.
 	TestRole *Role
+	// SkipCleanup prevents t.Cleanup from being invoked at the end of the test.
+	// Since this can result in large numbers of orphaned artifacts being left
+	// on the database server, it should only be used if another cleanup method
+	// is employed.
+	SkipCleanup bool
```

Comment:
Glad you implemented it this way, the default is `false` which makes this a backwards compatible change :)